app/routes/articles.js

```diff
@@ -12,7 +12,7 @@ const fs = require('fs')
 const path = require('path')
 const sharp = require('sharp')
 const JPEG = 'jpeg', JPEG_OPTIONS = {force: true, chromaSubsampling: '4:4:4'}
-
+const CATEGORIES_LIST = JSON.parse(require('fs').readFileSync(require('path').join(__dirname, '../staticdata/categories.json'), 'utf-8'))
```

Comment:
I believe you can require JSON files in NodeJS. Like `require('./someJSON.json')`.